src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,88 @@
+import { FC } from 'react';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import { Link } from 'react-router-dom';
+import { useFilters } from '../../hooks/useFilters';
+import cn from 'classnames';
+import { SORT_COLUMNS } from '../../constants/SORT_COLUMNS';
+
+type Props = {
+  visiblePeople: Person[];
+};
+
+export const PeopleTable: FC<Props> = ({ visiblePeople }) => {
+  const { sort, order, personSlug, handleSorting } = useFilters();
+
+  return (
+    <>
+      <table
+        data-cy="peopleTable"
+        className="table is-striped is-hoverable is-narrow is-fullwidth"
+      >
+        <thead>
+          <tr>
+            {SORT_COLUMNS.map(columnName => {
+              const lowerName = columnName.toLowerCase();
+
+              return (
+                <th key={columnName}>
+                  <span className="is-flex is-flex-wrap-nowrap">
+                    {columnName}
+                    <Link to={{ search: handleSorting(lowerName) }}>
+                      <span className="icon">
+                        <i
+                          className={cn('fas', {
+                            'fa-sort': sort !== lowerName,
+                            'fa-sort-up': sort === lowerName && !order,
+                            'fa-sort-down': sort === lowerName && order,
+                          })}
+                        />
+                      </span>
+                    </Link>
+                  </span>
+                </th>
+              );
+            })}
+            <th>Mother</th>
+            <th>Father</th>
+          </tr>
+        </thead>
+        <tbody>
+          {visiblePeople.map(person => {
+            return (
+              <tr
+                key={person.slug}
+                className={
+                  personSlug === person.slug ? 'has-background-warning' : ''
+                }
```

Comment:
it's better to use classnames in cases like that, it allows you to dynamically toggle classes based on state or props without manually concatenating strings, making the code more maintainable and easier to read.